src/components/style/forms/checkable.css

```diff
@@ -19,49 +19,7 @@
   top: 0;
 }
 
-.fdsCheckable label {
-  /* contains checkbox icon and text label */
-  display: inline-flex;
-  align-items: center;
-}
-
-.fdsCheckable .fds-icon {
-  color: var(--font-color-secondary);
-}
-
-.fdsCheckable input:focus + label svg {
+.fdsCheckable input:focus + .fdsCheckable--icon svg {
   box-shadow: inset 0 0 0 3px var(--border-color-focusRing);
   transition: box-shadow 250ms var(--motion-easing-default);
-}
-
-.fdsCheckable input:checked + label svg {
-  fill: var(--color-blue);
-}
-
-.fdsCheckable--disabled :matches(label,svg) {
-  color: var(--font-color-hint) !important;
-  fill: var(--font-color-hint) !important;
-}
-
-/**
- * Icon switching
- *
- * Both icons for checked/unchecked state are rendered in
- * the DOM; we use checked state selectors on the `input`
- * element to determine which one to show
- */
-.fdsCheckable input + label .fdsCheckable-icon--checked {
```

Comment:
`Radio` may still rely on this CSS

Reply:
Yep, looks like we may need to split up `fdsCheckable` between Radio and Checkbox until Radio is refactored